daemon/algod/api/algod.oas2.json

```diff
@@ -3647,6 +3647,10 @@
         },
         "exec-trace-config": {
           "$ref": "#/definitions/SimulateTraceConfig"
+        },
+        "initial-states": {
+          "description": "Allows returning initial state of applications before simulation.",
+          "type": "boolean"
```

Comment:
I'm inclined to think this should be part of `exec-trace-config`. I can sort of see why it isn't, it feels like something from "before" the trace. But it feels so intertwined from the standpoint of usage, that it would make more sense there.

Reply:
ok making it an option under `exec-trace-config`, yet another option? hopefully the last additional option '_>'

Reply:
actually, I had an idea, if we activate `state-changes` under simulate endpoint, can we also return the initial states of resources touched? I think this is somehow a reasonable move.

Reply:
I'd be ok with that. It seems like if you want changes, then you need the initial state.

Reply:
right, that was the intention: if we report the state changes, we probably want the (relevant) initial states to base off these diffs.

Making model changes now.